elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -604,4 +604,13 @@ defmodule Domain.Config.Definitions do
     default: nil,
     changeset: {Logo, :changeset, []}
   )
+
+  ##############################################
+  ## Sign-ups Enabled Flag
+  ##############################################
+
+  @doc """
+  Boolean flag to turn Sign-ups on/off.
+  """
+  defconfig(:sign_up_enabled, :boolean, default: true)
```

Comment:
I think we will add more of those so let's prefix the name and organize it.
```suggestion

  ##############################################
  ## Feature Flags
  ##############################################

  @doc """
  Boolean flag to turn Sign-ups on/off.
  """
  defconfig(:feature_sign_up_enabled, :boolean, default: true)
```

Reply:
Does that mean you'd like me to break out different flags for the up coming PR I'm working on rather than have a single `beta_features_enabled` flag?